src/reducers/example.tsx

```diff
@@ -0,0 +1,54 @@
+import { Reducer } from 'redux';
+
+const TOGGLE_ON = 'TOGGLE_ON';
```

Comment:
What do you think about using an `enum` for these? `enum`s are one of the things I really liked about TS when I was reading more about it.

Reply:
yep I also, think we should adopt ENUM for the redux actions.

Reply:
ok, I switched to `enum` for action types but it makes them longer to type. What's the advantage?